src/components/jobDetailsComponent/editJob.tsx

```diff
@@ -44,43 +59,32 @@ const EditJob: React.FC = () => {
     console.log(address);
   }, [address]);
 
-  const handleChange = (
-    e: React.ChangeEvent<HTMLInputElement>,
-    set: React.Dispatch<React.SetStateAction<string>>,
-    labelId: string
-  ): void => {
-    // control form field value
-    set(e.target.value);
-    // get the label
-    const label = document.getElementById(labelId);
-    // toggle class for label animation
-    /* eslint no-unused-expressions: ["error", { "allowTernary": true }] */
-    e.target.value
-      ? label.classList.add('active')
-      : label.classList.remove('active');
-  };
-
-  const isValid = (): boolean => {
-    const form = document.getElementById('createJobForm');
-    const validFields = form.querySelectorAll('.valid');
-    // 2 inputs + textarea are valid and address object returned from Google
-    return validFields.length === 2;
-  };
+  useEffect(() => {
+    // animate label for textarea to clone material-ui
+    const textarea: HTMLElement = document.getElementById('description');
+    const textareaLabel: HTMLElement = document.getElementById(
+      'descriptionLabel'
+    );
+    if (textarea.textContent.length) {
+      textareaLabel.classList.add('active');
+    }
+    textarea.addEventListener('focus', () => {
+      textareaLabel.classList.add('active');
```

Comment:
doesn't it apply active second time if  textarea.textContent.length is true too?

Reply:
i think the browser is smart enough to not do that